MultiPartyExample/MultiPartyViewController.swift

```diff
@@ -28,25 +27,25 @@ class MultiPartyViewController: UIViewController {
     var currentDominantSpeaker: TVIRemoteParticipant?
 
     // MARK: UI Element Outlets and handles
-    @IBOutlet weak var messageLabel: UILabel!
-    @IBOutlet weak var localParticipantVideoView: TVIVideoView!
-    @IBOutlet weak var localParticipantAudioIndicator: UIImageView!
+    @IBOutlet weak var containerView: UIView!
+    @IBOutlet weak var audioMuteButton: UIButton!
+    @IBOutlet weak var videoMuteButton: UIButton!
```

Comment:
This button appears when I use a simulator that doesn't have a camera. Can it be hidden or at least disabled/greyed out in this case?

Reply:
![Simulator Screen Shot - iPhone 6 - 2019-04-18 at 12 35 31](https://user-images.githubusercontent.com/1302577/56386386-79733600-61d6-11e9-882b-237a1c8e551c.png)

Reply:
Button is now disabled on simulator